uvm/servlets/admin/config/network/Interface.js

```diff
@@ -406,6 +406,17 @@ Ext.define('Ung.config.network.Interface', {
                     blankText: 'Address must be specified.'.t(),
                     vtype: 'ip4AddExcldDflt',
                     validator: function(value) {
+                        // Validation for not allowing network and broadcast address of selected netmask
+                        try {
+                            var staticPrefix = this.up('window').down('#intfcNetmask').getValue(),
+                                intfcNetMask = Util.getV4NetmaskMap()[staticPrefix];
+                            if(value == Util.getNetwork(value, intfcNetMask) || value == Util.getBroadcast(value, intfcNetMask)) {
+                                return Ext.String.format('Entered address is network or broadcast address of selected Netmask {0}.'.t(), intfcNetMask);
```

Comment:
Let's reword as "Address cannot be a network or broadcast address with netmask {0}"

Reply:
Updated the error message verbiage.
![Screenshot from 2024-03-14 20-33-20](https://github.com/untangle/ngfw_src/assets/154422821/e81a1bec-3d99-4141-9175-5ad0ed8aff88)